src/huggingface_hub/utils/_runtime.py

```diff
@@ -302,6 +303,11 @@ def is_google_colab() -> bool:
     return _is_google_colab
 
 
+def is_colab_enterprise() -> bool:
+    """Return `True` if code is executed in a Google Colab Enterprise environment."""
+    return os.environ.get("VERTEX_PRODUCT") == "COLAB_ENTERPRISE"
```

Comment:
maybe you could put this directly in `is_google_colab`? i.e. is_google_colab if on Colab but not on Colab Enterprise

Reply:
I'd rather keep it separated like this so that we have the correct information in the env dump (in case of a user reporting an issue)